styles.css

```diff
@@ -159,29 +174,28 @@ body {
   cursor: pointer;
 }
 
-.work > button:hover {
+.work button:hover {
```

Comment:
- [ ] As the previous code reviewer mentioned, please add the other button interactions like the pressed to completely match the Figma template button.
 - 
<img width="591" alt="image" src="https://user-images.githubusercontent.com/61919617/230084680-eeb7bf6e-c19e-441a-920b-db64467f9891.png">